python/py-pyqt5/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup               cxx11  1.1
 
 name                    py-pyqt5
 version                 5.12.1
-revision                0
+revision                1
```

Comment:
The `revision` should stay at 0 because you are adding a new port, not actually changing an existing port. There is need for people to rebuild `py-pyqt5`.

Reply:
I was initially planning on changing the ```webengine``` variant to install the ```py-pyqt5-webengine``` port, but that resulted in a circular dependency (or something) and didn't work. Right now, you're right that it doesn't need a revision increase; will change that.